renovate.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+  ],
+  "enabledManagers": [
+    "custom.regex"
+  ],
+  "packageRules": [
+    {
+      "groupName": "all non-major dependencies",
+      "groupSlug": "all-minor-patch",
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "prBodyColumns": [
+        "Package",
+        "Type",
+        "Update",
+        "Change",
+        "Pending"
+      ]
+    },
+    {
+      "groupName": "all major dependencies",
+      "groupSlug": "all-major",
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "prBodyColumns": [
+        "Package",
+        "Type",
+        "Update",
+        "Change",
+        "Pending"
+      ]
+    }
+  ],
+  "customManagers": [
+    {
+      "description": "ModSecurity 3",
+      "depNameTemplate": "ModSecurity 3",
+      "customType": "regex",
+      "fileMatch": [
+        "^docker-bake.hcl$"
+      ],
+      "matchStrings": [
+        "^variable \"modsec3-version\" {\\n\\s+default = \"(?<currentValue> [^\"]+)\""
+      ],
+      "packageNameTemplate": "owasp-modsecurity/ModSecurity",
+      "datasourceTemplate": "github-releases"
```

Comment:
Is it smart enough to use the proper version? E.g. upgrading the corresponding major?

Reply:
Yes, with the additional changes I've made.